Gemfile

```diff
@@ -17,6 +17,7 @@ gem 'jquery-rails'
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
 
+gem 'acts-as-taggable-on', '~> 4.0'
```

Comment:
Any reason why we don't use the latest version 5.0?

Reply:
We will change to version 5.0.